mix.exs

```diff
@@ -1,20 +1,57 @@
-defmodule Membrane.Caps.Audio.Opus.Mixfile do
-  use Mix.Project
-
-  def project do
-    [app: :membrane_caps_audio_opus,
-     version: "0.0.1",
-     elixir: "~> 1.3",
-     description: "Membrane Multimedia Framework (Audio.Opus caps)",
-     maintainers: ["Marcin Lewandowski"],
-     licenses: ["LGPL"],
-     name: "Membrane Caps: Audio.Opus",
-     source_url: "https://bitbucket.org/radiokit/membrane-caps-audio-opus",
-     preferred_cli_env: [espec: :test],
-     deps: deps()]
-  end
-
-
-  def application, do: []
-  defp deps, do: []
-end
+defmodule Membrane.Opus.Format.Mixfile do
+  use Mix.Project
+
+  @version "0.2.0"
+  @github_url "https://github.com/membraneframework/membrane_opus_format"
+
+  def project do
+    [
+      app: :membrane_opus_format,
+      version: @version,
+      elixir: "~> 1.10",
+      deps: deps(),
+
+      # hex
+      description: "Membrane Multimedia Framework (Opus Audio Format Description)",
+      package: package(),
+
+      # docs
+      name: "Membrane: Opus Format",
+      source_url: @github_url,
+      homepage_url: "https://membraneframework.org",
+      docs: docs()
+    ]
+  end
+
+  def application do
+    [
+      extra_applications: []
+    ]
+  end
+
+  defp deps do
+    [
+      {:membrane_core, "~> 0.5.0", only: :dev}
```

Comment:
I don't think it's needed here. It was added somewhere to be able to reference Core modules in docs (when it's a dependency, ExDoc adds links to proper docs)

Reply:
I have added it because the formatter configuration inherits from core's one

Reply:
1. `only: :dev` means formatting will fail in env other than dev and our CI configs will run format under test environment
2. core in .formatter.exs is needed only when using one of the macros defined there (like `def_options`) so it is not needed